### ISSUE TITLE
suggestion- exclude picking up specified items

### ISSUE THREAD

**Issue by bomba1749** [maruohon/tweakeroo#226]
it would be really useful if there was a tweak that made it so that you wouldn't pick up certain items. This tweak would greatly reduce the amount of inventory management needed while playing Minecraft, and overall increase efficiency.
 For example, this tweak could have application in mining, so that you would automatically pick up ores, but not stones. 
Another use would be to  quickly sort items, you could break a chest, without even needing to know the contents, and you would pick up all of the items you want, and leave all of the unwanted items.

**Comment by maruohon** [comment]
It's not possible to prevent picking up items with a client-side mod, except in single player.

The mod could automatically throw away all of the "blacklisted" items from the player's inventory, but that does have the side effect that the items will be  moved, ie. be thrown out like you would normally throw out items to the player's looking direction. And also even if you were to manually take some of those items from a chest, the mod would then immediately throw them out. (Unless it has a more complicated code for tracking stacks that the player takes manually inside an inventory screen.)

So overall I'm not sure if I wold want to implement this at all for multiplayer, due to the above mentioned issues. I think they would make the feature very clunky and awkward and more annoying than useful, maybe...

**Comment by bomba1749** [comment]
yeah, if it isnt able to work in multiplayer, then it would probably take more effort to make than it would save in use
